### PR TITLE
state-card: handle if more-info = hidden

### LINKED ISSUE #460: aesthetic: state-card also has cursor:pointer if domain is listed in HIDE_MORE_INFO
https://github.com/home-assistant/home-assistant-polymer/blob/19187ce518190823a8ccc5e7fc3d262cd218fa74/src/cards/ha-entities-card.html#L18

https://github.com/home-assistant/home-assistant-polymer/blob/master/src/util/hass-util.html#L34

### PR DESCRIPTION
**Pull request by c727**
this is untested. also I'm not sure if I should use 2x dom-if instead?

should fix #460 and #463

### CODE REVIEW

**Comment by c727** [comment]
ok, I got dev environment running.

Have to get the on-Tab working

**Comment by c727** [comment]
should be OK now. maybe the names for vars and functions are not the best

**Comment by andrey-git** [comment]
What's wrong with the previous tap approach?

**Comment by c727** [comment]
@andrey-git 
well I linked the 2 issues maybe my bug descriptions are not the best....
let's take input_text as example: it has more-info-card = hidden but still has cursor:pointer which gives the impression that it has an on-click event. 2nd is that every state-card (also if more-info-card = hidden) has this on-tap listener so input_text can still show an more-info-card if you click on the outer pixels of the statecard. the 2nd one is even more annoying if you use custom state-cards which don't use full width.

the approach in this PR is to add the on-tap dyanamic in ready() but it's not working cause my queryselector is not ok.

this approach https://github.com/home-assistant/home-assistant-polymer/compare/master...c727:patch-5 is using dom-if but I don't like to have the code for the div twice

File: src/cards/ha-entities-card.html [hunk 1]
```diff
@@ -129,6 +135,15 @@
 
     return canToggleCount > 1;
   }
+
+  ready() {
+    //super.ready();
+    var CardsWithMore = this.$.querySelectorAll('.state .has-more-info-card');
```
Comment (andrey-git) [inline src/cards/ha-entities-card.html]: I think this.$ is a map to unconditional children with IDs.
Try `this.querySelectorAll`

Comment (c727) [reply src/cards/ha-entities-card.html]: CardsWithMore.length is also 0

**Pull request closed without merging**